src/commcare_cloud/ansible/roles/formplayer/templates/application.properties.j2

```diff
@@ -65,3 +65,8 @@ server.forward-headers-strategy=NATIVE
 {% if formplayer_enable_cache|default(False) %}
 spring.cache.type=caffeine
 {% endif %}
+
+{% if formplayer_detailed_tagging_domains is defined %}
+detailed_tagging.domains="{{ formplayer_detailed_tagging_domains|join('", "') }}"
+detailed_tagging.tag_names="{{ formplayer_detailed_tags|join('", "') }}"
```

Comment:
@calellowitz here's what I settled on. I preemptively did the same for `detailed_tagging.domains`, but wouldn't be surprised if that ends up formatted differently.

Reply:
So when either list is empty it'll result in one entry of an empty string on the formplayer side. I'm trying to justify this by telling myself these _shouldn't_ be empty, but the best solution seems to be to include a check on the list's length before formatting this way, otherwise leave it empty.